dashboard/test/ui/features/support/connect.rb

```diff
@@ -34,19 +35,15 @@ def saucelabs_browser(test_run_name)
   }
 
   sauce_options[:priority] = ENV['PRIORITY'].to_i if ENV['PRIORITY']
-  capabilities["sauce:options"] ||= {}
-  capabilities["sauce:options"].merge!(sauce_options)
-
-  very_verbose "DEBUG: Capabilities: #{CGI.escapeHTML capabilities.inspect}"
```

Comment:
I did not preserve this `very_verbose` method call; it is here for the purposes of extra debug logging, and as far as I can tell has not been useful for quite some time. Happy to put it back in if folks would prefer; just let me know

Reply:
This is the type of print statement I think is honestly easier just to insert yourself if/when you need it, custom to your specific needs.